bigtable/metric_util.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2024 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bigtable
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	btpb "google.golang.org/genproto/googleapis/bigtable/v2"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/proto"
+)
+
+const (
+	defaultCluster = "unspecified"
+	defaultZone    = "global"
+)
+
+// get GFE latency in ms from response metadata
+func getServerLatency(headerMD metadata.MD, trailerMD metadata.MD) (float64, error) {
```

Comment:
Since this function gets the server latency from response metadata and isn't a getter, could you rename this to something like `getServerLatencyFromResponse`

Reply:
Renamed to extractServerLatency